featurebyte/common/utils.py

```diff
@@ -420,3 +424,132 @@ def _repr_html_(self) -> str:
             '<div style="margin:30px; padding: 20px; border:1px solid #aaa">'
             f"{highlighted_code}</div>"
         )
+
+
+class InfoDict(Dict[str, Any]):
+    """
+    Featurebyte asset information dictionary that can be displayed in HTML
+    """
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        self.class_name = data.pop("class_name", "Unknown")
+        super().__init__(data)
+
+    def _repr_html_(self) -> str:
```

Comment:
Tested on `feature.info()`, encountered following error:
![Screenshot 2023-05-16 at 9 42 34 AM](https://github.com/featurebyte/featurebyte/assets/37915749/9896f08d-0eca-4384-b498-015127427225)

Reply:
Seems like it does not handle properly when the list is empty. Below is the example of the feature info dictionary:
```
{'name': 'CustomerSpendingStability_7d28d',
 'created_at': '2023-05-16T01:35:13.757000',
 'updated_at': None,
 'entities': [{'name': 'grocerycustomer',
   'serving_names': ['GROCERYCUSTOMERGUID'],
   'catalog_name': 'Grocery V31 - playground (spark)'}],
 'primary_entity': [{'name': 'grocerycustomer',
   'serving_names': ['GROCERYCUSTOMERGUID'],
   'catalog_name': 'Grocery V31 - playground (spark)'}],
 'tables': [{'name': 'INVOICEITEMS',
   'status': 'PUBLIC_DRAFT',
   'catalog_name': 'Grocery V31 - playground (spark)'},
  {'name': 'GROCERYINVOICE',
   'status': 'PUBLIC_DRAFT',
   'catalog_name': 'Grocery V31 - playground (spark)'}],
 'default_version_mode': 'AUTO',
 'version_count': 1,
 'catalog_name': 'Grocery V31 - playground (spark)',
 'dtype': 'FLOAT',
 'primary_table': [{'name': 'INVOICEITEMS',
   'status': 'PUBLIC_DRAFT',
   'catalog_name': 'Grocery V31 - playground (spark)'}],
 'default_feature_id': '6462dd287b12204e7e1c2ba6',
 'version': {'this': 'V230516', 'default': 'V230516'},
 'readiness': {'this': 'DRAFT', 'default': 'DRAFT'},
 'table_feature_job_setting': {'this': [{'table_name': 'GROCERYINVOICE',
    'feature_job_setting': {'blind_spot': '3600s',
     'frequency': '3600s',
     'time_modulo_frequency': '60s'}}],
  'default': [{'table_name': 'GROCERYINVOICE',
    'feature_job_setting': {'blind_spot': '3600s',
     'frequency': '3600s',
     'time_modulo_frequency': '60s'}}]},
 'table_cleaning_operation': {'this': [], 'default': []},
 'versions_info': None,
 'metadata': {'input_columns': {'Input0': {'data': 'INVOICEITEMS',
    'column_name': 'GroceryProductGuid',
    'semantic': None},
   'Input1': {'data': 'INVOICEITEMS',
    'column_name': 'TotalCost',
    'semantic': None}},
  'derived_columns': {},
  'aggregations': {'F0': {'name': 'CustomerSpending_7d',
    'column': 'Input1',
    'function': 'sum',
    'keys': ['GroceryCustomerGuid'],
    'window': '7d',
    'category': 'GroceryProductGuid',
    'filter': False},
   'F1': {'name': 'CustomerSpending_28d',
    'column': 'Input1',
    'function': 'sum',
    'keys': ['GroceryCustomerGuid'],
    'window': '28d',
    'category': 'GroceryProductGuid',
    'filter': False}},
  'post_aggregation': {'name': 'CustomerSpendingStability_7d28d',
   'inputs': ['F0', 'F1'],
   'transforms': ['cosine_similarity']}}}

```